src/main/asciidoc/java/http.adoc

```diff
@@ -1251,24 +1251,24 @@ client.websocket("/some-uri", websocket -> {
 
 ==== Writing messages to WebSockets
 
-If you wish to write a single binary WebSocket message containing a single WebSocket frame to the WebSocket (a
-common case) the simplest way to do this is to use `link:../../apidocs/io/vertx/core/http/WebSocket.html#writeMessage-io.vertx.core.buffer.Buffer-[writeMessage]`:
+If you wish to write a single binary WebSocket message to the WebSocket you can do this with
+`link:../../apidocs/io/vertx/core/http/WebSocket.html#writeBinaryMessage-io.vertx.core.buffer.Buffer-[writeBinaryMessage]`:
```

Comment:
Well it should answer to the question "which method should I used when I want to send a single message to the websocket ?"

In this section you mention 'writeBinaryMessage', but later you mention 'writeFinalBinaryFrame'. You this second method be mentioned here too ?